config/api.yml

```diff
@@ -2081,6 +2081,8 @@
         :identifier: ems_infra_edit
       - :name: resume
         :identifier: ems_infra_edit
+      - :name: refresh
```

Comment:
this is off, we already have a refresh action, maybe your intent was to add the identifier to the refresh above on line 2076, similar to what query does to allow for 2 identifiers. so they can so a refresh if they have the ems_infra_refresh or ems_cloud_refresh role.  In which case we have the refresh test, we could add another test to make sure refresh works with ems_cloud_refresh too.

Reply:
I moved to here after a comment by Jason. Should I put it underneath `identifier: ems_infra_refresh`

Yes my intent was to allow a user to refresh if they have either `ems_infra_refresh or ems_cloud_refresh role`

Reply:
yes please, that's the intent of the multiple identifiers in the actions list (user must have at least one such role).